aten/src/ATen/CMakeLists.txt

```diff
@@ -175,9 +175,13 @@ file(GLOB mem_eff_attention_cuda_cu "native/transformers/cuda/mem_eff_attention/
 file(GLOB mem_eff_attention_cuda_kernels_cu "native/transformers/cuda/mem_eff_attention/kernels/*.cu")
 file(GLOB mem_eff_attention_cuda_cpp "native/transformers/cuda/mem_eff_attention/*.cpp")
 
+# flash_attn for cuda requires excessive memory, so restrict parallelism in ninja builds
+add_library(flash_attention_cuda OBJECT)
+set_property(GLOBAL PROPERTY JOB_POOLS flash_attention_pool=3)
```

Comment:
> This is done with a ninja job pool. I've hard-coded the number of jobs to 3, since 16GB of memory sounds reasonably standard.

16 GB sounds like a lot to me, if it's only for compiling this single component, compiling in serial sounds safer.

Reply:
can we make this configurable?

Reply:
Would be curious to see how much of a difference the overall build times change between 1 job and 3 jobs.